translations/user.create.yml

```diff
@@ -2,36 +2,30 @@ description: 'Crea usuarios en el sitio'
 help: 'El comando <info>user:create</info> le ayuda a crear usuarios.'
 welcome: 'Bienvenido al generador de usuarios Drupal'
 options:
-  username: 'Nombre del usuario que será creado'
-  password: 'Contraseña del usuario'
-  roles: 'Roles del usuario'
-  email: 'E-mail del usuario'
-  status: 'Estado del usuario'
+    username: 'Nombre del usuario que será creado'
+    password: 'Contraseña del usuario'
+    roles: 'Roles del usuario'
+    email: 'E-mail del usuario'
+    status: 'Estado del usuario'
 questions:
-  username: 'Nombre del usuario que será creado'
-  password: 'Contraseña del usuario (dejar vacío para autogenerar)'
-  roles: 'Roles de usuario (dejar vacío para omitir)'
-  email: 'E-mail del usuario (dejar vacío para omitir)'
-  status: 'Estado del usuario (dejar vacío para omitir)'
+    username: 'Nombre del usuario que será creado'
+    password: 'Contraseña del usuario (dejar vacío para autogenerar)'
+    roles: 'Roles de usuario (dejar vacío para omitir)'
+    email: 'E-mail del usuario (dejar vacío para omitir)'
+    status: 'Estado del usuario (dejar vacío para omitir)'
 messages:
-  user-id: 'ID de usuario'
-  username: 'Nombre de usuario'
-  password: 'Contraseña'
-  email: 'E-mail'
-  roles: 'Roles'
-  created: 'Creado'
-  status: 'Estado'
-  user-created: 'El usuario "%s" ha sido creado con éxito'
+    user-id: 'ID de usuario'
+    username: 'Nombre de usuario'
+    password: Contraseña
+    email: E-mail
+    roles: Roles
+    created: Creado
+    status: Estado
+    user-created: 'El usuario "%s" ha sido creado con éxito'
 examples:
-  - description: 'Crear usuario especificando nombre de usuario, contraseña, rol, email y estado'
-    execution: |
-      drupal user:create  john p455w0rd  \
-        --roles='authenticated'  \
-        --email="john@anexusit.com"  \
-        --status="1"
-  - description: 'Crear usuario administrador especificando nombre de usuario, contraseña, rol, email y estado'
-    execution: |
-      drupal user:create  doe p455w0rd  \
-        --roles='administrator'  \
-        --email="doe@anexusit.com"  \
-        --status="1"
+    -
+        description: 'Crear usuario especificando nombre de usuario, contraseña, rol, email y estado'
+        execution: "drupal user:create  john p455w0rd  \\\n  --roles='authenticated'  \\\n  --email=\"john@anexusit.com\"  \\\n  --status=\"1\""
```

Comment:
Hi @novia713 

In order to provide a better output in example section, please don't remove the \ in behalf of \\\n